bot/exts/fun/movie.py

```diff
@@ -107,7 +119,7 @@ async def genres(self, ctx: Context) -> None:
         """Show all currently available genres for .movies command."""
         await ctx.send(f"Current available genres: {', '.join('`' + genre.name + '`' for genre in MovieGenres)}")
 
-    async def get_movies_data(self, client: ClientSession, genre_id: str, page: int) -> list[dict[str, Any]]:
+    async def get_movies_data(self, client: ClientSession, genre_id: str, page: int) -> tuple[dict[str, Any], int]:
```

Comment:
I assume `resp.json()` is still a list of dictionaries so this should be:
```suggestion
    async def get_movies_data(self, client: ClientSession, genre_id: str, page: int) -> tuple[list[dict[str, Any]], int]:
```

Reply:
Yep mb.